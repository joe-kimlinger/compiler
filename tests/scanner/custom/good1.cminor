// integers
x: integer;
integer: 50023e342
array [3] integer
{1, 2, 3, 5}
