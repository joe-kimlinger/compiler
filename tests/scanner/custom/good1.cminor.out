COMMENT
IDENTIFIER x
COLON
INTEGER
SEMICOLON
INTEGER
COLON
INTEGER_LITERAL 50023
IDENTIFIER e342
ARRAY
LSQBRACKET
INTEGER_LITERAL 3
RSQBRACKET
INTEGER
LBRACKET
INTEGER_LITERAL 1
COMMA
INTEGER_LITERAL 2
COMMA
INTEGER_LITERAL 3
COMMA
INTEGER_LITERAL 5
RBRACKET
